src/controllers/applicationController.ts

```diff
@@ -0,0 +1,127 @@
+import * as auth from '../middleware/authMiddleware';
+import {User} from '../entity/User';
+import {Application} from '../entity/Application';
+
+import {getManager} from 'typeorm';
+import {Request, Response} from 'express';
+import {validate} from 'class-validator';
+import app from '../server';
+
+import * as path from 'path';
+import * as fs from 'fs';
+
+const RESUME_ROOT = __dirname + '/../../resumes/';
```

Comment:
What does this path translate to when the server is running? Is it a `resumes` directory in the root of the app?
This may also be something we want to make configurable.

Reply:
Yeah, that's correct. I'll change it so that it takes from the .env file, and defaults to that directory if one isn't given.